pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs

```diff
@@ -1343,59 +1392,82 @@ private void UpdateForm()
         private bool isAssociated; // True if the current transition list contains associated proteins
 
         private void checkBoxAssociateProteins_CheckedChanged(object sender, EventArgs e)
+        {
+            UpdateProteinAssociationState(AssociateProteinsMode.preview, out _);
+        }
+
+        private void UpdateProteinAssociationState(AssociateProteinsMode mode, out bool canceled)
```

Comment:
It's unusual for a "void" function to have an out parameter.
Normally, you would just change return type of this function to be "bool" and get rid of the out parameter.